qa/suites/upgrade/pacific-x/stress-split/2-first-half-tasks/rbd_api.yaml

```diff
@@ -1,9 +1,17 @@
 meta:
 - desc: |
    librbd C and C++ api tests
+overrides:
+  ceph:
+    log-ignorelist:
+      - overall HEALTH_
+      - \(CACHE_POOL_NO_HIT_SET\)
+      - \(POOL_APP_NOT_ENABLED\)
+      - is full \(reached quota
+      - \(POOL_FULL\)
 first-half-tasks:
 - workunit:
-     branch: octopus
+     branch: pacific
```

Comment:
I don't think this PR should be changing any branch names and I don't remember seeing this when I looked initially.  Is this an intentional change or just a rebase issue?

Reply:
I did include this as I noticed other yaml files in this tree were set to `branch: pacific`. See `qa/suites/upgrade/pacific-x/parallel/workload/test_rbd_api.yaml` . It appeared it needed to be set to the branch it was testing. If this isn't the case what is the branch config option used for?

Reply:
> I did include this as I noticed other yaml files in this tree were set to `branch: pacific`. See `qa/suites/upgrade/pacific-x/parallel/workload/test_rbd_api.yaml` .

You may have a point, but it certainly shouldn't be done in the same commit.

> It appeared it needed to be set to the branch it was testing. If this isn't the case what is the branch config option used for?

@yuriw Could you please answer Chris' question?

Reply:
> You may have a point, but it certainly shouldn't be done in the same commit.
Yes you are right. Upon further thought, I suggest we remove this change completely from this scope and create a new issue depending on the response from @yuriw.